ocfweb/docs/docs/staff/procedures/setting-up-lacp.md

```diff
@@ -0,0 +1,77 @@
+[[!meta title="Setting up bridging and link aggregation"]]
+
+Bridging and LACP will be configured through puppet, but these intructions should
+serve as background and to help debug.
```

Comment:
It could be worth writing a paragraph explaining what all this means, and perhaps define some terms as well.